src/compojure/api/meta.clj

```diff
@@ -592,12 +599,18 @@
                           :letks []
                           :responses nil
                           :middleware []
-                          :swagger {}
+                          :info {}
                           :body body}
                          options)
 
         static? (not (or dynamic? (route-args? route-arg) (seq lets) (seq letks)))
 
+        static-context? (and static? context?)
+        info (cond-> info
+               static-context? (assoc ::static-context? static-context?))
```

Comment:
should this be non-qualified as it's "a system value". qualified could be left for user extensions...?